### ISSUE TITLE
Strong tiling artifacts

### ISSUE THREAD

**Issue by Buglakova** [hci-unihd/plant-seg#190]
When I run the network prediction, the result has strong tiling artifacts. I use quite big halo, but it doesn't help. I encountered this before when using U-Net that wasn't trained for long enough or didn't see enough ground truth, which is kind of the case when applying a pretrained network to different data. As a workaround, would be nice to have an option for smooth transition between tiles, like in the original U-Net publication, where within the halo the weight of the tile in the result falls linearly towards the edge of the tile.

Here is an example:
![Screenshot from 2024-02-05 13-49-59](https://github.com/hci-unihd/plant-seg/assets/89460016/7e4eba43-568f-409a-8798-1da07a026ae2)
![Screenshot from 2024-02-05 13-50-18](https://github.com/hci-unihd/plant-seg/assets/89460016/54736bec-7e35-446a-a563-6a32c3487ff0)

Furthermore, it propagates to the superpixels (here GASP run on the wrong channel, but still you can see clearly the squares)
![Screenshot from 2024-02-05 13-50-43](https://github.com/hci-unihd/plant-seg/assets/89460016/48348324-d141-48b8-8a46-a1d8da0e9c11)

**Comment by qin-yu** [comment]
I'm looking into it and believe it relates to #205

**Comment by qin-yu** [comment]
Fixing #205 doesn't help, i.e. changing size of halo doesn't improve prediction, which indicates more serious problems. Thus #220 and https://github.com/wolny/pytorch-3dunet/pull/113

Progress:

1. Halo implementation is fixed, which already greatly improved the prediction (https://github.com/wolny/pytorch-3dunet/pull/113#issue-2236219588)
2. Intensity normalisation doesn't cause problem at least for ovules (training data) and mouse (another dataset).
3. Batch norm fix hallucinations created by group norm in mouse embryo (I mean you need to train with batch norm again).

**Comment by qin-yu** [comment]
For PlantSeg:

- Halo implementation and intensity normalisation outside networks are fixed by #220 
- Batch norm with default `track_running_stats = True` should be used in models instead of group norm
